OrbitGl/CaptureWindow.cpp

```diff
@@ -734,11 +743,11 @@ void CaptureWindow::RenderHelpUi() {
       "Measure: \"Right Click + Drag\"\n"
       "Toggle Help: 'H'";
 
-  const uint32_t kHelpMessageFontSize = 2 * font_size_;
   Vec2 text_bounding_box_pos;
   Vec2 text_bounding_box_size;
-  text_renderer_.AddText(help_message, world_x, world_y, GlCanvas::kZValueTextUi,
-                         Color(255, 255, 255, 255), kHelpMessageFontSize, -1.f /*max_size*/,
+  float string_height = text_renderer_.GetStringHeight(help_message, font_size_);
+  text_renderer_.AddText(help_message, world_x, world_y - string_height, GlCanvas::kZValueTextUi,
```

Comment:
To avoid this world_y - string_height calculation and getting the height before, a function like AddTextGivenTopLimit or something like that, don't you like?

Reply:
Yes good idea, I'd add a parameter to specify the origin of the text (kBaseLine, kTopLine, kBottomLine)

Reply:
Will do in subsequent pr.